src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -94,7 +91,12 @@ public static class TagHelperDescriptorFactory
 #if !DNXCORE50
             if (designTime)
```

Comment:
Just to clarify. With pre-compilation `designTime` will always be `false` right?